### DISCUSSION TITLE
Mouse thinks it is on the other monitor

### DISCUSSION THREAD

**Discussion by gabikv** [i3/i3#5724] [Q&A]
I have two monitors - the one on the right is called DP-1 and the one on the left is called HDMI-1. I use Ubuntu by the way. Moving the mouse from one side to the other works correctly because I ran this command in the terminal:
```
xrandr --output DP-1 --auto --right-of HDMI-1
```

I wrote the exact thing in the ~/.config/i3/config but with `exec --no-startup-id` before. So It should load the same command but it doesn't. 

Now the mouse thinks it is on the other monitor and highlights the window on the exact same position but on the wrong monitor!

When switching windows with the mod + arrow keys the command didn't change anything.

When I gave up and then after some hours it magically fixed itself but when restarting it went back to the same problem.

Conclusion:
When the mouse is on the left monitor it thinks it is on the right monitor on the same position so it's very hard to for example click on small buttons. BTW I have used `mod + shift + c` and `mod + shift + r` but it doesn't reload and fix the problem.

I think it is that the windows are rendered on the wrong monitor because when it magically fixes it self every window has switched monitor.

How can I fix this problem with the windows rendering on the wrong monitor?

**Comment by Charlie-Steer** [comment]
I'm having the same issue on Linux Mint.
